### ISSUE TITLE
All functionality of the screen is not operable through a keyboard (Tab) interface Admin issue

### ISSUE THREAD

**Issue by ramdev1980** [percussion/percussioncms#802]
**Describe the bug**
All the options (User Created Sites and pages, all controls/options under Workflow, Roles, Users and Categories Tab are not operable through a keyboard interface (tab) Admin issue

**To Reproduce**

1. Go to Admin (http://localhost:9992/cm/app/?view=workflow&site=test3&mode=readonly&name=index.html&path=%2FSites%2Ftest3%2Findex.html&pathType=page)
2. Click on a site.
3. See that all the options/controls on the screen, under  Workflow, Roles, Users and Categories  are not accessible using Tab key.

**Expected behavior**
All page elements should be operable through a keyboard interface (tab).

**Additional context**
Navigate all the options available for similar issue and fix.

**Comment by ramdev1980** [comment]
Fixed with https://github.com/percussion/percussioncms/issues/781
